config/routes.rb

```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get "/home", to: "home#index"
+
+  use_doorkeeper do
+    skip_controllers :authorizations, :applications, :authorized_applications
+  end
+
+  unless defined?(::Rake::SprocketsTask)
```

Comment:
Do you have any issues around here?
Can we dismiss this line out? 🤔

Reply:
Yes! I can't seem to deploy this to Production, I found this as a potential fix for Production x.x and I have seemed to have lost the link for where I saw this 😅